crates/typst-render/src/lib.rs

```diff
@@ -387,6 +392,44 @@ fn render_outline_glyph(
         let scale = text.size.to_f32() / text.font.units_per_em() as f32;
 
         let mut pixmap = None;
+
+        let rule = sk::FillRule::default();
```

Comment:
@Dherse could you review the stroking code here? I think you are most familiar with how the transforms work and how arguments to `to_sk_paint` should be configured.

Reply:
Sure, I'll make a local copy and have a looksy 👍

Reply:
@Enter-tainer The only mistake is that `on_text` should be set to `true` instead of `false` and then they'll render correctly :)